example/src/plugins/job-ui-single/pages/JobForm.tsx

```diff
@@ -0,0 +1,121 @@
+import {
+  EBlueprint,
+  TJob,
+  Stack,
+  useBlueprint,
+  Loading,
+  TAttribute,
+  EntityPickerButton,
+  JobStatus,
+  BlueprintPicker,
+  TJobHandler,
+  TJobWithRunner,
+} from '@development-framework/dm-core'
+import { Button, TextField } from '@equinor/eds-core-react'
+
+import React, { useState } from 'react'
+
+/**
+ * A component for creating an entity of type Job.
+ *
+ * This component depends on the Job blueprint in dm-job. If the blueprint changes, this component might have to be updated.
+ * The form consists of regular input fields for primitive blueprint attributes, but 'runner' and 'applicationInput' must be handled differently.
+ * The user must select type for the 'runner' with a BlueprintPicker, and the user must select a reference to use as 'applciationInput'.
+ *
+ * @param onSubmit Function to run when Job is submitted.
+ * @param jobRunnerType Type of job runner to add to the Job entity (for example dmss://DemoDataSource/apps/MySignalApp/models/SignalGeneratorJob)
+ * @param applicationInputType Type of applicationInput to add to the Job entity.
+
+ */
+export const JobForm = (props: {
+  onSubmit: (job: TJob) => void
+  applicationInputType: string
+  jobRunnerType: string
+}) => {
+  const defaultJobValues: TJobWithRunner = {
+    type: EBlueprint.JOB,
+    runner: { type: props.jobRunnerType } as TJobHandler,
+    status: JobStatus.NotStarted,
+    started: 'Not started',
+  }
+  const [formData, setFormData] = useState<TJobWithRunner>(defaultJobValues)
+  const {
+    blueprint: jobBlueprint,
+    isLoading: isBlueprintLoading,
+    error,
+  } = useBlueprint(EBlueprint.JOB)
+
+  if (isBlueprintLoading) {
+    return <Loading />
+  }
+  if (error) {
+    throw new Error('Could not load Job blueprint')
+  }
+  return (
+    <div>
+      <Stack spacing={1}>
+        {jobBlueprint.attributes.map((attribute: TAttribute) => {
```

Comment:
I think you should have a filtering here. It does not make sense to let the user be able to edit the job status or the type imo. You could also filter away the optional attributes to avoid the empty return on L101